tests/unit/async_/io/test_class_bolt4x3.py

```diff
@@ -230,7 +230,7 @@ async def test_hint_recv_timeout_seconds(
     sockets = fake_socket_pair(address,
                                packer_cls=AsyncBolt4x3.PACKER_CLS,
                                unpacker_cls=AsyncBolt4x3.UNPACKER_CLS)
-    sockets.client.settimeout = mocker.AsyncMock()
+    sockets.client.settimeout = mocker.Mock()
```

Comment:
shouldn't it remain an `AsyncMock` since we're in an async unit test?

Reply:
Nope. That's because the method is a sync one (non-blocking) even for our async socket wrapper. Here's the definition: https://github.com/neo4j/neo4j-python-driver/blob/cc76ac5a2bb6c5933e757adee5f2415a831c78ff/neo4j/_async_compat/network/_bolt_socket.py#L143